src/widgets/modals/SearchForm.js

```diff
@@ -0,0 +1,159 @@
+/* eslint-disable react/forbid-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2020
+ */
+
+import React, { useMemo, useState } from 'react';
+import { View, StyleSheet, FlatList, Text } from 'react-native';
+import PropTypes from 'prop-types';
+
+import { FormControl } from '../FormControl';
+
+import { getFormInputConfig } from '../../utilities/formInputConfigs';
+
+import { APP_FONT_FAMILY, DARK_GREY } from '../../globalStyles';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import { queryPatientApi, queryPrescriberApi } from '../../utilities/network/lookupApi';
+
+const RECORD_TYPES = {
+    PATIENT: 'patient',
+    PRESCRIBER: 'prescriber',
+}
+
+const FORM_CONFIGS = {
+    [RECORD_TYPES.PATIENT]: 'searchPatient',
+    [RECORD_TYPES.PRESCRIBER]: 'searchPrescriber',
```

Comment:
I think putting this `FORM_CONFIGS` in with the `formInputConfigs`, then if the `dataSource` is just one of these constants I think it would be a lot simpler.

Reply:
Good point, I do think that would be a nicer way to do it, but probably best for a separate issue as it will touch other components (I'm thinking one issue to put all the constants into `formInputConfigs` and update all components with references to e.g. `'patient'` to use them).

Reply:
Updated epic #2446 with related issue.